contracts/erc20-bridge-sampler/test/mainnet_tests.ts

```diff
@@ -0,0 +1,39 @@
+import { blockchainTests, describe, expect, toBaseUnitAmount } from '@0x/contracts-test-utils';
+import { BigNumber } from '@0x/utils';
+
+import { artifacts } from './artifacts';
+import { ERC20BridgeSamplerContract } from './generated-wrappers/erc20_bridge_sampler';
+
+blockchainTests.fork.skip('Mainnet Sampler Tests', env => {
+    let testContract: ERC20BridgeSamplerContract;
+    before(async () => {
+        testContract = await ERC20BridgeSamplerContract.deployFrom0xArtifactAsync(
+            artifacts.ERC20BridgeSampler,
+            env.provider,
+            { ...env.txDefaults, from: '0x6cc5f688a315f3dc28a7781717a9a798a59fda7b' },
+            {},
+        );
+    });
+
+    describe('sampleSellsFromCurve()', () => {
+        const curveAddress = '0x2e60CF74d81ac34eB21eEff58Db4D385920ef419';
+        const daiTokenIdx = new BigNumber(0);
+        const usdcTokenIdx = new BigNumber(1);
+
+        it('samples sells from Curve DAI->USDC', async () => {
+            const samples = await testContract
+                .sampleSellsFromCurve(curveAddress, daiTokenIdx, usdcTokenIdx, [toBaseUnitAmount(1)])
+                .callAsync();
+            expect(samples.length).to.be.bignumber.greaterThan(0);
+            expect(samples[0]).to.be.bignumber.greaterThan(0);
```

Comment:
I guess one of the things I get paranoid about is whether these endpoints are expressing the correct units. So maybe we can check that `samples[0] > 0 && samples[0] < toBaseUnitAmount(2, 6)`